pkg/http-server/file-scan.go

```diff
@@ -93,6 +95,18 @@ func (g *APIHandler) scanFile(w http.ResponseWriter, r *http.Request) {
 	// severity is the minimum severity level of violations that the user want to get informed about: low, medium or high
 	severity := r.FormValue("severity")
 
+	// read config_only from the form data
+	configOnlyValue := r.FormValue("config_only")
+	if configOnlyValue != "" {
+		configOnly, err = strconv.ParseBool(configOnlyValue)
+		if err != nil {
+			errMsg := fmt.Sprintf("error while reading 'config_only' value. error: '%v'", err)
```

Comment:
May be we can return the incorrect value received as part of the request, so that the caller understands his mistake.

Reply:
Yes, that is part of the err.

Reply:
Ok, makes sense